active_learning_multiannotator/active_learning.ipynb

```diff
@@ -0,0 +1,721 @@
+{
```

Comment:
don't forget to stick some standard naming convention for the scores throughout the notebook and the helper files (in variable names, docstrings, text, etc). I believe we wanted to use name: "active learning quality scores"